README.md

```diff
@@ -1,2 +1,173 @@
-# XamExpertDay2021
-Repository for material/slides of the XamExpertDay2021
+<img src="_Images_/XamExpertDay2021.png" width="800" />
+
+
+
+# Xamarin Expert Day 2021 (@XamExpertDay)
+
+All slides and relevant things from the XamExpertDay2021. The event was held virtually on October 1st  2021.
+
+https://expertday.forxamarin.com/
```

Comment:
URL is wrong and does not work. Please use https://www.xamexpertday.com

Reply:
done